stylesheet.css

```diff
@@ -594,3 +632,553 @@ textarea {
+    list-style: none;
+    order: 0;
+    width: 10vw;
+    padding: 0;
+    height: 90vh;
+    margin: 0;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .social-icon {
+    margin: 0 0 2vh 0;
+  }
+
+  .content {
+    display: flex;
+    order: 1;
+    max-width: fit-content;
+    padding-left: 0;
+  }
+
+  .greeting {
+    padding: 0;
+    font-family: 'Crete Round', serif;
+    font-size: 4.5rem;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 5.09375rem;
+  }
+
+  .presentation {
+    font-family: 'Inter', sans-serif;
+    font-size: 2rem;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 2.75rem;
+  }
+
+  .description {
+    font-family: 'Inter', sans-serif;
+    font-size: 1.25rem;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 2rem;
+    max-width: 45vw;
+  }
+
+  .works {
+    max-width: 100vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+    min-height: fit-content;
+  }
+
+  .works_intro {
+    display: flex;
+    flex-direction: row;
+    width: 80vw;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 6vh;
+  }
+
+  .recent_works {
+    font-family: 'Crete Round', serif;
+    font-size: 6vh;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 6.5vh;
+    margin-right: 2vw;
+    margin-bottom: 0;
+    justify-content: flex-start;
+    width: fit-content;
+  }
+
+  .recent_works_break {
+    flex: 0.95;
+  }
+
+  .main_project {
+    display: flex;
+    flex-direction: row;
+    width: 80vw;
+    align-items: flex-start;
+    justify-content: space-between;
+  }
+
+  .project_image {
+    display: flex;
+    width: 50vw;
+  }
+
+  .content_project_div {
+    display: flex;
+    flex-direction: column;
+    width: 28vw;
+    padding: 0, 0, 0, 2vw;
+  }
+
+  .work_title_main {
+    margin: 0 0 1.5rem 0;
+    font-family: 'Crete Round', serif;
+    font-size: 5vh;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 5.5vh;
+  }
+
+  .work_description_main {
+    margin-bottom: 1.5rem;
+    font-family: 'Inter', sans-serif;
+    font-size: 2.5vh;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 3vh;
+  }
+
+  .tools_list_main {
+    margin: 0;
+    max-width: 30vw;
+  }
+
+  .tool_main {
+    font-family: 'Inter', sans-serif;
+    font-size: 1.5vh;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 1.2rem;
+    padding: 1vh 2vw;
+  }
+
+  .see_project_main {
+    margin-top: 1.5rem;
+    max-width: fit-content;
+  }
+
+  .projects {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    width: 80vw;
+    margin: 0;
+    height: fit-content;
+  }
+
+  .project_card {
+    display: flex;
+    flex-direction: column;
+    width: 26vw;
+    height: 50vh;
+    margin-bottom: 2vh;
+    padding: 0;
+    align-items: flex-start;
+    justify-content: flex-end;
+  }
```

Comment:
Please kindly implement hover effect for the project cards as suggested earlier. When a user hover around the card the design below supposed to pop-up (project image and the see project button); by default the project button you added should not be there. 👇 

<img width="138" alt="Untitled" src="https://github.com/ommarz/Portfolio/assets/103386183/157f2543-f000-4698-bdb5-006032e75a7b">